docs/template/public/kss.less

```diff
@@ -1,4 +1,4 @@
-@import url(https://fonts.googleapis.com/css?family=Dosis:400,300,600,700);
+@import url(https://fonts.googleapis.com/css?family=Dosis:400,500,600);

```

Comment:
I doubt we need to include that at all here, as we have the techne from distribution folder included?